db2x_manxml fails on non-ascii chars

In maintainer mode, db2x_manxml from docboox2x-0.8.8 fails to convert conky-1.10.0's docs.mxml to a man page because it contains non-ascii characters. In fact, db2x_manxml runs `iconv -futf8 -tascii` as a subroutine and iconv itself fails on invalid characters.

I'm not sure what should be done about this. Removing the characters (see patch below) allows the manpage to be generated.

Here is the error message:

/usr/bin/iconv: illegal input sequence at position 7849
/usr/bin/db2x_manxml: program in pipeline exited with an error
doc/CMakeFiles/docs.dir/build.make:51: recipe for target 'doc/docs.html' failed
make[2]: *** [doc/docs.html] Error 1
CMakeFiles/Makefile2:218: recipe for target 'doc/CMakeFiles/docs.dir/all' failed

And the obvious patch:

```
diff -aur old/doc/config_settings.xml new/doc/config_settings.xml
--- old/doc/config_settings.xml	2015-06-21 14:49:06.000000000 -1000
+++ new/doc/config_settings.xml	2015-06-29 10:05:00.147318479 -1000
@@ -80,7 +80,7 @@
         </term>
         <listitem>A comma-separated list of strings to use as the bars of a graph output
         to console/shell. The first list item is used for the minimum bar height and the
-        last item is used for the maximum. Example: " ,_,▁,▂,▃,▄,▅,▆,▇,█".
+        last item is used for the maximum.
         <para /></listitem>
     </varlistentry>
     <varlistentry>
```

--- a/cmake/Docbook.cmake
+++ b/cmake/Docbook.cmake
@@ -72,7 +72,7 @@ if(MAINTAINER_MODE)
 			COMMAND ${APP_XSLTPROC} http://docbook.sourceforge.net/release/xsl/current/html/docbook.xsl ${CMAKE_CURRENT_SOURCE_DIR}/${FIL}.xml > ${CMAKE_CURRENT_SOURCE_DIR}/${FIL}.html
 
 			COMMAND ${APP_DB2X_XSLTPROC} -s man ${CMAKE_CURRENT_SOURCE_DIR}/${FIL}.xml -o ${FIL}.mxml
-			COMMAND ${APP_DB2X_MANXML} ${FIL}.mxml --output-dir ${CMAKE_CURRENT_SOURCE_DIR}
+			COMMAND ${APP_DB2X_MANXML} --encoding=utf-8 ${FIL}.mxml --output-dir ${CMAKE_CURRENT_SOURCE_DIR}
 			COMMAND ${APP_XSLTPROC} http://docbook.sourceforge.net/release/xsl/current/html/docbook.xsl ${CMAKE_CURRENT_SOURCE_DIR}/${FIL}.xml > ${CMAKE_CURRENT_SOURCE_DIR}/${FIL}.html
 			COMMAND ${APP_MAN} -P '${APP_LESS} -is' ${CMAKE_CURRENT_SOURCE_DIR}/${PROJECT_NAME}.1 > ${CMAKE_SOURCE_DIR}/README
 			COMMAND ${APP_SED} -i "s/[[:cntrl:]]\\[[0-9]*m//g" ${CMAKE_SOURCE_DIR}/README

